### PR TITLE
Spanish Translation Account form and New Password

### PR DESCRIPTION
**Pull request by oruburos**
I have verified that this pull request:

* [x] has no linting errors (`npm run lint`)
* [x] is from a uniquely-named feature branch and has been rebased on top of the latest master. (If I was asked to make more changes, I have made sure to rebase onto master then too)
* [x] is descriptively named 

This PR introduces Spanish Translation for the Account Form/View, ApiKeyForm and List and New Password Form/View

### CODE REVIEW

**Comment by andrewn** [comment]
This is looking very good.

@oruburos Can you fix the merge conflicts in this branch and then let me know when it's done please?

**Comment by oruburos** [comment]
@andrewn Hi, I've merged the most recent develop branch with this PR to solve the conflicts, can you have a look please?

File: client/modules/User/components/APIKeyForm.jsx [hunk 1]
```diff
@@ -39,7 +39,8 @@ class APIKeyForm extends React.Component {
   }
 
   removeKey(key) {
-    const message = `Are you sure you want to delete "${key.label}"?`;
+    // const message = `Are you sure you want to delete "${}"?`;
+    const message = i18next.t('APIKeyForm.ConfirmDelete', { key_label: key.label });
```
Comment (andrewn) [inline client/modules/User/components/APIKeyForm.jsx:43]: This should use `withTranslation`, not `i18next` as it's within a component.

Comment (oruburos) [reply client/modules/User/components/APIKeyForm.jsx]: Ok, the change requested it is complete, can you check this please?

Comment (andrewn) [reply client/modules/User/components/APIKeyForm.jsx]: You also need to remove the `i18next` import from the top of the file as it's no longer used. The lint should be warning about this.

Comment (oruburos) [reply client/modules/User/components/APIKeyForm.jsx]: Ok, the import is deleted, can you have a look please.?

**Pull request merged**